### PR TITLE
Add shields to project review tickets to promote use.

### PR DESCRIPTION
**Pull request by mattbk**
Closes https://github.com/gratipay/gratipay.com/issues/4634.

### CODE REVIEW

**Comment by mattbk** [comment]
https://github.com/gratipay/gratipay.com/blob/master/tests/py/test_project_review_process.py

**Comment by mattbk** [comment]
Next commit will have a test, but I need to fix the trailing slash before `/svg` in both code and test.

**Comment by mattbk** [comment]
:tada:

**Comment by mattbk** [comment]
Now onto `/about/me/widgets`...

**Comment by mattbk** [comment]
> Now onto `/about/me/widgets`...

This should actually go on the project, not the owner.

**Comment by mattbk** [comment]
image

**Comment by mattbk** [comment]
image

File: templates/team-base.html [hunk 1]
```diff
@@ -62,6 +62,7 @@
     {% set pages = [ ('/',              _('Profile'))
                    , ('/history/',      _('History'))
                    , ('/receiving/',    _('Receiving'))
+                   , ('/widgets/',    _('Widgets'))
```
Comment (whit537) [inline templates/team-base.html]: 💃

Comment (mattbk) [reply templates/team-base.html]: Good? Should I use Shields instead?

Comment (whit537) [reply templates/team-base.html]: No I think this is good.

**Comment by whit537** [comment]
Rebased (was 9676a908efd555faf8d00f5d5a1a7093a774e22f), and then I did a little formatting of both code and resulting content in abdad0722c8f17e10d447b9bb536051b2fafe4e9.

Issue example: https://github.com/whit537/test/issues/4.

Widget page:

screen shot 2017-09-28 at 8 51 11 am

**Comment by whit537** [comment]
I'm ready for this to be merged if you're good with my commit, @mattbk.

**Comment by mattbk** [comment]
I think it looks good.

**Comment by whit537** [comment]
Y u no like, Travis?

**Comment by whit537** [comment]
Rebased and Travis should be fixed, was 6b25f2c34174241880242e55c3ea787fff950d4a.

**Pull request merged**